runtime/common/src/claims.rs

```diff
@@ -151,6 +151,8 @@ decl_error! {
 		PotUnderflow,
 		/// A needed statement was not included.
 		InvalidStatement,
+		/// The account already has a vested balance.
+		VestedBalanceExists,
```

Comment:
You created a new event and now `DestinationVesting` is unused in the code

Reply:
Is this one better? Or should I revert to `DestinationVesting`?

Reply:
i mean, doesn't really matter to me. But should get rid of one and use the other :)